app/templates/components/ui-table/cell/events/view/tickets/attendees/cell-order.hbs

```diff
@@ -13,12 +13,12 @@
         {{t 'Payment via'}} {{extraRecords.paidVia}}
       </span>
     {{/if}}
-    {{#if (eq extraRecords.status 'completed')}}
-      {{moment-format extraRecords.completedAt 'MMMM Do YYYY, h:mm A -'}}
-      {{moment-from-now extraRecords.completedAt}}
+    {{#if (eq record.status 'completed')}}
```

Comment:
Why reverting extraRecords ?

Reply:
in the same file {{ record.status }} return `completed` i.e. the status of order for attendee. so if  codition should be like - 
```
{{#if (eq record.status 'completed') }}
```
`extraRecords.status` is literaly returning `nothing`

Reply:
That's because model is attendee and record order, extraRecords wont work.

Reply:
exactly